example/index.js

```diff
@@ -50,74 +50,73 @@ const login = function (request, reply) {
     if (request.method === 'get' ||
         message) {
 
-        return reply('<html><head><title>Login page</title></head><body>' +
+        return '<html><head><title>Login page</title></head><body>' +
             (message ? '<h3>' + message + '</h3><br/>' : '') +
             '<form method="post" action="/login">' +
             'Username: <input type="text" name="username"><br>' +
             'Password: <input type="password" name="password"><br/>' +
-            '<input type="submit" value="Login"></form></body></html>');
+            '<input type="submit" value="Login"></form></body></html>';
     }
 
     const sid = String(++uuid);
-    request.server.app.cache.set(sid, { account: account }, 0, (err) => {
 
-        if (err) {
-            return reply(err);
-        }
+    await request.server.app.cache.set(sid, { account }, 0);
+    request.cookieAuth.set({ sid });
 
-        request.cookieAuth.set({ sid: sid });
-        return reply.redirect('/');
-    });
+    return h.redirect('/');
 };
 
-const logout = function (request, reply) {
+const logout = function (request, h) {
 
     request.cookieAuth.clear();
-    return reply.redirect('/');
+    return h.redirect('/');
 };
 
-const server = new Hapi.Server();
-server.connection({ port: 8000 });
+const server = new Hapi.Server({ port: 8000 });
```

Comment:
`const server = Hapi.server({ port: 8000 });`

Reply:
Are we not using `new` anymore? I thought I saw that somewhere.. maybe I was reading the wrong docs version!

Reply:
https://hapijs.com/api/17.0.1#-serveroptions
compared to:
https://hapijs.com/api/16.6.2#new-serveroptions

And it looks like this is where it was changed to a factory method:
https://github.com/hapijs/hapi/commit/f35b8c5c8a1958f3a3eb9462b4a1ad537a28fe57#diff-c945a46d13b34fcaff544d966cffcabaR23

Reply:
Cool, thanks for the research!